templates/article.html

```diff
@@ -23,6 +23,11 @@ <h1>{{ article.title }}</h1>
             {% csrf_token %}
             {% block script %}
               {{ block.super }}
+
+              <script>
+              /* Fixes a bug in django-tinymce with TINYMCE_INCLUDE_JQUERY = False, see https://github.com/aljosa/django-tinymce/issues/234 */
+              var django = {jQuery: jQuery};
```

Comment:
We don't define this is base so we avoid problems everywhere?

Reply:
The execution order is kinda fragile here, so I wanted to put it right before `init_tinymce.js` to avoid problems in the future if somebody reorders the scripts. And I don't want entire tinymce in base to avoid loading it when not needed (old ckeditor also did this but in a more hacky way)

A better fix may be to copy `init_tinymce.js` from django-tinymce and modify the lines that call jQuery.noConflict but that on the other hand may break when the plugin is updated

Reply:
Let's separate this into a template maybe?

Reply:
If anything I would make this a JS file, but a JS file for one line seemed unnecessary